### PR TITLE
Update mount

### PR DESCRIPTION
**Pull request by xiangce**
- Remove the `regex` and use `split` instead
- Refine the related classes
- Keep compatible with the original version
- fix: #2059
- fix: #2069

### CODE REVIEW

File: insights/specs/insights_archive.py [hunk 1]
```diff
@@ -46,7 +46,6 @@ class InsightsArchiveSpecs(Specs):
     docker_info = simple_file("insights_commands/docker_info")
     docker_list_containers = simple_file("insights_commands/docker_ps_--all_--no-trunc")
     docker_list_images = simple_file("insights_commands/docker_images_--all_--no-trunc_--digests")
-    dumpdev = simple_file("insights_commands/awk_.ext_234_._print_1_.proc.mounts")
```
Comment (bfahr) [inline insights/specs/insights_archive.py]: @xiangce I believe it's OK to remove this spec since it doesn't appear in the uploader JSON.

**Comment by bfahr** [comment]
This depends on [insighs-core-assets PR 227](https://github.com/RedHatInsights/insights-core-assets/pull/227).

**Comment by JoySnow** [comment]
It's good to me.

**Comment by csams** [comment]
@xiangce @bfahr @ppandey-redhat Hey everybody, let's lean away from the "pretend to be a dict" or dynamic attribute setting mixins and just use regular attributes initialized in `__init__`.  See my [comment on the related issue](https://github.com/RedHatInsights/insights-core/issues/2081#issuecomment-513773022).

**Comment by xiangce** [comment]
Hey all, since the update to the `MountOpts` will impact the `FStab` too, I just reverted all the update to the `MountOpts` and `MountEntry` and merge this PR at first. And open another PR to resolve the #2081

**Comment by xiangce** [comment]
The following updates are in #2104

**Pull request merged**